File: lib/es6/belt_Int.js



function fromString(i) do
  i$1 = parseInt(i, 10);
  if (isNaN(i$1)) then do
    return ;
  end else do
    return i$1;
  end end 
end end

export do
  fromString ,
  
end
--[[ No side effect ]]

File: lib/js/belt_Int.lua
__console = {log = print};


function fromString(i) do
  i_1 = __parseInt(i, 10);
  if (__isNaN(i_1)) then do
    return ;
  end else do
    return i_1;
  end end 
end end

exports = {};
exports.fromString = fromString;
return exports;
--[[ No side effect ]]
